node.gyp

```diff
@@ -379,11 +379,13 @@
         'src/async_wrap-inl.h',
         'src/base_object.h',
         'src/base_object-inl.h',
+        'src/callback_scope.h',
         'src/connection_wrap.h',
         'src/connect_wrap.h',
         'src/debug_utils.h',
         'src/env.h',
         'src/env-inl.h',
+        'src/exceptions.h',
```

Comment:
These two headers don’t exist anymore, I think we want to remove them from this PR?

Reply:
Done.